tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -26,8 +26,21 @@ using ::testing::ElementsAreArray;
 
 class BasePoolingOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BasePoolingOpModel(BuiltinOperator type, const TensorData& input,
```

Comment:
For function parameters, could you use const reference instead of passing-by-value here?

Reply:
@haozha111 , thanks for pointing this out, i have made the changes as suggested by you, kindly check and approve.